### ISSUE TITLE
Including the parse transform seems to export all functions

### ISSUE THREAD

**Issue by eproxus** [williamthome/doctest#46]
When using the include file like so:

```erlang
-ifdef(TEST).
-include_lib("doctest/include/doctest.hrl").
-endif.
```

it seems that the header file exports all functions in the current module. I'm running xref on my code in the test profile (to include the test usage of the code in the analysis) and it complains about all private functions being exported but not used (which is true).

Is there a way the include file could avoid exporting all private functions in the module?

**Comment by williamthome** [comment]
Yeah, that behavior should be changed. All functions are exported to test all docs, but unexported functions are unavailable to users, so testing them does not make sense. I wanted to change that some time ago, but I'm giving open-source a lower priority, but I can take a look at this in the next few days.

**Comment by eproxus** [comment]
Don't stress it if you don't have the time or energy 🙏

**Comment by williamthome** [comment]
Fixed in [v0.10.0](https://hex.pm/packages/doctest/0.10.0).

**Comment by S0AndS0** [comment]
On the off-chance future readers that happen across this also happen to want to test private functions, then `-export` such things within the `-ifdef(TEST).` block, ex;

```erlang
-ifdef(TEST).
-include_lib("doctest/include/doctest.hrl").
-export([private_funk/0]).
-endif.
```
